modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/template/RESTClientTemplateContextContributor.java

```diff
@@ -59,6 +61,17 @@ public RESTClient(
 		}
 
 		public Object get(String path) throws Exception {
+			try {
+				return _get(path);
+			}
+			catch (Throwable throwable) {
+				_log.error(throwable, throwable);
+
+				throw throwable;
+			}
+		}
+
+		private Object _get(String path) throws Exception {
```

Comment:
Could we do it inline? I am not sure about creating this method

Reply:
I did it inline to avoid shifting all the code to the left. A classic `buy space` from Brian.